.env.docker

```
@@ -4,18 +4,42 @@ NODE_ENV=dev
+SYNAPSE_DB_SSL=false
+
+# Session sync between kratos and synapse
+SESSION_SYNC_ENABLED=true
+SESSION_SYNC_INTERVAL_MS=300000
+KRATOS_DB_DATABASE=kratos
+
+
+# Ory Hydra Configuration
+HYDRA_SYSTEM_SECRET=d6fbf37acaf143b9b4c7560094bb9af237e3bd50e7afe7c7888a8948d0938398
+# Public base URL (used for URLS_SELF_PUBLIC, URLS_SELF_ISSUER, and constructing login/consent URLs)
+HYDRA_PUBLIC_URL=http://localhost:3000
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The HYDRA_PUBLIC_URL key should go before the HYDRA_SYSTEM_SECRET key

(UnorderedKey)